src/user/dashboard/settings/styles/settings.scss

```diff
@@ -2,7 +2,7 @@
   display: flex;
   min-height: 100vh;
   height: auto;
-  font-family: Muli, sans-serif;
+  font-family: Inter;
```

Comment:
Just a suggestion, better to add fallback font family as well.

Reply:
Yeah, I forgot to do that. Thanks for reminding.